### PR TITLE
Created wercker step to check cd gating.

### PR DESCRIPTION
**Pull request by theazureshadow**
This is the public Wercker step to check our continuous deployment gating flags in s3. I've tested the shell command itself, but don't know if there's a way to test a Wercker step without actually putting it into their system. Tips appreciated :)

/cc @christophercurrie @downie

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+ENVIRONMENT_FLAG="$(curl $ENVIRONMENT_URL 2>/dev/null)"
+SERVICE_FLAG="$(curl $SERVICE_URL 2>/dev/null)"
+
+if [ $ENVIRONMENT_FLAG == "off" ]; then
+  echo "Deployments to $WERCKER_CD_GATE_ENVIRONMENT are off";
+  exit 1
+fi
+if [ $SERVICE_FLAG == "off" ]; then
+  echo "Deployments of $WERCKER_CD_GATE_SERVICE are off";
```
Comment (downie) [inline run.sh]: This might be overkill, but maybe this message should include the environment?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Checks web service for continuous deployment flags per environment and service name.
+
+## Options
+
+* `endpoint` - root url for environment/service http calls
+* `environment` - name of environment (e.g. `production`)
+* `service` - name of service
```
Comment (downie) [inline README.md]: We might want to standardize our language. In the robbie scripts, we refer to an environment and an application, not a service. I don't care which we use, but I'd like it to be consistent. Thoughts?

**Comment by downie** [comment]
minor comments. LGTM.
I'm also not sure how to test this without trying it. Maybe just go for it?

File: README.md (hunk 2 above)
Comment (theazureshadow) [inline README.md]: Wercker uses "App". Good call.

**Comment by theazureshadow** [comment]
@downie: I've tested this (and made a couple fixes) using a fork of this step and a test Wercker project. Merging.

**Pull request merged**